examples/arm/qemu-shell.nix

```diff
@@ -0,0 +1,11 @@
+let
+  pkgs = (import ./arm-cross.nix).pkgs;
+  crossPkgs = pkgs.pkgsCross.aarch64-multiplatform;
+in
+pkgs.mkShell {
+  LANG="C.UTF-8";
+  LD_LIBRARY_PATH = "${crossPkgs.buildPackages.gcc-unwrapped}/aarch64-unknown-linux-gnu/lib64";
```

Comment:
I'm wondering why this is needed and there's no corresponding `RUNPATH` entry generated. Something worth looking into outside of this PR.

Reply:
Created an issue here: https://github.com/tweag/rules_haskell/issues/1521